website/frontend/styles/ApiSection.scss

```diff
@@ -1,12 +1,15 @@
 @import "lib/variables";
+@import "lib/colors";
 
 .Apisection {
     display: flex;
+    justify-content: center;
     align-items: center;
-    margin: 58px $mainPadding;
+    max-width: $columnWidth;
+    margin: 168px auto 147.86px auto;
```

Comment:
In the future, we will need to round off these. We still to agree on the round off approach though.